src/komponenter/TelefonnummerInput/TelefonnummerInput.tsx

```diff
@@ -26,6 +26,7 @@ const TelefonnummerInput: React.FunctionComponent<Props> = props => {
 
     return (
         <Input
+            bredde="S"
```

Comment:
Ser ut som dette gjør at telefonnummer feltet blir kortere enn alle andre felt. Ved å ikke spesifisere noen bredde her vil de ta opp så mye plass som de har, i tilfelle en hel `<Column md="6">` i `DeltakerInfoDel`.
Ser at de blir for lange i ArbeidsgiverInfoDel og også for nav, men her er det satt `<Column md="12">` for telefonnummer..